docs/snippets/README.md

```diff
@@ -0,0 +1,3 @@
+# WooCommerce Code Snippets
```

Comment:
```suggestion
# WooCommerce code snippets
```

We should use sentence case for the title, as mentioned on https://github.com/woocommerce/woodocs/blob/0d037666cbf2fc8056c89a1b67de3b7fe08c895a/style-guide.md?plain=1#L16-L17